Sources/PubNub/Helpers/Crypto/CryptoModule.swift

```diff
@@ -273,6 +275,48 @@ public struct CryptoModule {
     return streamDecryptionResult
   }
 
+  /// Decrypts the stream from the given URL and writes it to the output path
+  ///
+  /// - Parameters:
+  ///   - url: URL of the encrypted stream
+  ///   - outputPath: Path to write the decrypted stream
+  /// - Returns:
+  ///  - **Success**: A decrypted `InputStream` object
+  ///  - **Failure**: `PubNubError` describing the reason of failure
+  @discardableResult
+  public func decryptStream(from url: URL, to outputPath: URL) -> Result<InputStream, PubNubError> {
```

Comment:
Is there old decrypt Method that The `result` variable is of type `InputStream`. Should it get deprecation message?

Reply:
We would like to treat this method as a convenient method to already existing method responsible for decoding a stream. It's up to you as a customer which one you would like to pick.

It's likely we will deprecate existing `decrypt(stream: contentLength: to:)` method. However, I don't wanna do it in a hurry and consider first all the pros and cons before doing it